src/Controller/StudentController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class StudentController extends AbstractController
+{
+    /**
+     * @Route("/student", name="student", )
+     * @param Request $request
+     * @return Response
+     */
+    public function index(Request $request): Response
+    {
+        $name = $request->get('name') ?: "Nenurodytas" ;
```

Comment:
```suggestion
        $name = $request->get('name',  "Nenurodytas" );
```
`get` metodas turi antrą parametrą. Tai verta išnaudoti standartines Symfony funkcijas.

Reply:
fixed